lib/ts/recipe/emailpassword/components/features/signInAndUp/index.tsx

```diff
@@ -284,15 +287,15 @@ function getThemeSignUpFeatureFormFields(formFields: NormalisedFormField[], reci
                     const emailExists = (
                         await recipe.recipeImpl.doesEmailExist({
                             email: value,
-                            config: recipe.config,
+                            config: recipe.webJsRecipe.config,
                             userContext: {},
                         })
                     ).doesExist;
 
                     if (emailExists) {
                         return "EMAIL_PASSWORD_EMAIL_ALREADY_EXISTS";
                     }
-                } catch (_) {}
+                } catch {}
```

Comment:
If this is of type general error, then return that custom message. Else return undefined (like we are doing now)